bindings/src/test/java/io/pravega/storage/filesystem/FileSystemLockTest.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.storage.filesystem;
+
+import io.pravega.common.io.FileHelpers;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.Random;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Unit tests for FileSystemBasedLock.
+ */
+public class FileSystemLockTest {
+
+    static final long ONE_HOUR = Duration.of(1, ChronoUnit.HOURS).toMillis();
+    static final long ONE_MILLI = Duration.of(1, ChronoUnit.MILLIS).toMillis();
+    static final long TWO_MILLI = Duration.of(2, ChronoUnit.MILLIS).toMillis();
+    static final long HUNDRED_MILLI = Duration.of(100, ChronoUnit.MILLIS).toMillis();
+
+    private File baseDir = null;
+    private AtomicBoolean inCriticalSection = new AtomicBoolean();
```

Comment:
I'm not sure I understand why this is a class variable since it is only used in one test case.